src/modules/fw_att_control/fw_att_control_main.cpp

```diff
@@ -963,129 +963,27 @@ FixedwingAttitudeControl::task_main()
 				float yaw_manual = 0.0f;
 				float throttle_sp = 0.0f;
 
-				/* Read attitude setpoint from uorb if
-				 * - velocity control or position control is enabled (pos controller is running)
-				 * - manual control is disabled (another app may send the setpoint, but it should
-				 *   for sure not be set from the remote control values)
-				 */
-				if (_vcontrol_mode.flag_control_auto_enabled ||
-				    !_vcontrol_mode.flag_control_manual_enabled) {
-					/* read in attitude setpoint from attitude setpoint uorb topic */
-					roll_sp = _att_sp.roll_body + _parameters.rollsp_offset_rad;
-					pitch_sp = _att_sp.pitch_body + _parameters.pitchsp_offset_rad;
-					yaw_sp = _att_sp.yaw_body;
-					throttle_sp = _att_sp.thrust;
-
-					/* reset integrals where needed */
-					if (_att_sp.roll_reset_integral) {
-						_roll_ctrl.reset_integrator();
-					}
-
-					if (_att_sp.pitch_reset_integral) {
-						_pitch_ctrl.reset_integrator();
-					}
-
-					if (_att_sp.yaw_reset_integral) {
-						_yaw_ctrl.reset_integrator();
-						_wheel_ctrl.reset_integrator();
-					}
-
-				} else if (_vcontrol_mode.flag_control_velocity_enabled) {
-
-					/* the pilot does not want to change direction,
-					 * take straight attitude setpoint from position controller
-					 */
-					if (fabsf(_manual.y) < 0.01f && fabsf(_roll) < 0.2f) {
```

Comment:
@AndreasAntener Do you have any what the use of the roll constraint is? This was the cause of the wing rock phenomena.